lib/version_utils.pm

```diff
@@ -131,8 +131,12 @@ sub sle_version_at_least {
     my ($version, %args) = @_;
     my $version_variable = $args{version_variable} // 'VERSION';
 
+    if ($version eq '12') {
+        return !check_var($version_variable, '11-SP4');
+    }
+
     if ($version eq '12-SP1') {
-        return !check_var($version_variable, '12');
+        return sle_version_at_least('12', version_variable => $version_variable) && !check_var($version_variable, '12');
     }
```

Comment:
Could you give more verified tests to prove this modified `sle_version_at_least`?

Reply:
http://10.67.17.147/tests/2015#step/patch_before_migration/22